app/assets/stylesheets/michigan-benefits/organisms/_main-footer.scss

```diff
@@ -0,0 +1,14 @@
+.main-footer {
+  font-weight: 300;
+}
+
+.main-footer {
+  .illustration--cfa, .illustration--civilla {
+    float: left;
+    margin-top: 46px;
```

Comment:
The specificity here  is justifiable, in my opinion, because it's referencing a very specific component. If this was something that was used across pages and across contexts, I would totally agree with you.

If this was extracted out to a variable it would be specific to this component - which would push the value down one level into a variable named *for* this component.